ocaml/idl/datamodel_certificate.ml

```diff
@@ -32,6 +39,12 @@ let t =
     ~messages_default_allowed_roles:_R_READ_ONLY
     ~contents:
       [ uid   _certificate ~lifecycle
+      ; field ~qualifier:DynamicRO ~lifecycle:[Published, rel_next, ""]
+          ~ty:String "name"
+          "The name of the certificate, only present on certificates of type 'ca'"
+      ; field ~qualifier:DynamicRO ~lifecycle:[Published, rel_next, ""]
```

Comment:
I would guess that existing certs should have type ```ca`` by default? Though I'm not sure what name we would give them (perhaps just their uuid)

Reply:
Existint certs in the In 8.2 Only host certificates are present in the patabase they should be of type `host